src/main/kotlin/racingcar/domain/RacingGame.kt

```diff
@@ -0,0 +1,29 @@
+package racingcar.domain
+
+class RacingGame(
+    private val numberOfRaces: Int,
+    private val racingCars: RacingCars
+) {
+
+    init {
+        require(numberOfRaces >= MINIMUM_NUMBER_OF_RACES) { "경주 횟수는 최소 $MINIMUM_NUMBER_OF_RACES 이상이어야 합니다." }
+    }
+
+    fun play(): GameRecord {
+        val raceRecords = ArrayList<RaceRecord>()
```

Comment:
```suggestion
        val raceRecords = mutableListOf<RaceRecord>()
```
처럼 사용할 수도 있을 거 같아요. `mutableListOf` 내부 구현은 `ArrayList`로 되어있기 때문에 결과는 같지만 코틀린이 한 층 추상화해서 제공하는 방식을 사용해보는 건 어떨까요?